### ISSUE TITLE
Card ends up in wrong column or swimlane when dragging

### ISSUE THREAD

**Issue by emahuni** [kanboard/kanboard#4026]
I have noticed this many times when dragging. The card ends up in the wrong column partly because the drop area is limited and there is no quick visual feedback when dragging over a long column.

Solution may be to highlight the entire column where dropping will occur in addition to the dashed card placeholder. This gives the user a feel where the card will go regardless of the position the card will be in the column.

**Comment by creecros** [comment]
The dotted lines show where it is about to drop. Don't think you needed three issues for all these, and my guess is your on a mobile device. Switch to desktop mode on mobile, and it might improve your experience.

![image](https://raw.githubusercontent.com/creecros/Bak2topbotm/master/screenshot/test_drag.gif)

**Comment by emahuni** [comment]
I used three issues to address different issues.

I think you miss-understood me. In your above example if that source column was very long such that you had to scroll down to the task you are dragging, moving that issue to the short target column isn't easy.

That's where the highlight whole column comes in.

Let me enable desktop mode as you have suggested thanks.

**Comment by creecros** [comment]
Try it on desktop mode, then report back. You should find that the dotted lines will appear where you are dragging, regardless of how long that column is.

**Comment by creecros** [comment]
Also, I agree with you on the column headers, would be nice if they didn't scroll out of view.

**Comment by emahuni** [comment]
> Try it on desktop mode, then report back. You should find that the dotted lines will appear where you are dragging, regardless of how long that column is.

Ummm it doesn't, I don't know whether this is a mobile browser issue or something, I did it in desktop mode, oh and desktop mode changes a lot of things by the way :+1: 
 Could you show it in a capture as you did above?

**Comment by creecros** [comment]
![image](https://raw.githubusercontent.com/creecros/empty/master/test_drag_columns.gif)

**Comment by emahuni** [comment]
oh i see where you are not getting me.

Do this, scroll your long column to the bottom until you can see the short column and drag a task from the long column to the short column. Since the short column is scrolled out of view, there are no dotted lines shown. 

This is not an error, it's a design issue, whereby one doesn't know where they are dropping the task.

**Comment by creecros** [comment]
@emahuni @rafacamargo123 

I get what he is saying now.

won't make a difference what version you use, he is right, it's by design. 

would need to check if `draggable-placeholder` element is visible, and if not, maybe highlight the columns background color or something like that, to show which column it is being dragged to.

**Comment by rafacamargo123** [comment]
@creecros yes but the fact the placeholder is not shown shouldn’t make the card end up in the wrong column.. prior to v1.2.6 you had the problem where your card sometimes disappeared, went to other swimlanes or columns, solved on issue #3625. I asked him if he had updated his kanboard because on current version, even without placeholder, you are sure your card will end up in te column you are hovering it with.

**Comment by creecros** [comment]
Gotcha, I've never been able reproduce that portion. Figured he meant, that it went in the wrong column because he couldn't see where it is going.

**Comment by tuxayo** [comment]
can this issue be closed? @emahuni

**Comment by ChangePlaces** [comment]
there are jquery / javascript solutions to create 'sticky' headers for columns when they scroll out of view. this wouldn't be easy as the headers would have to swap over as you scroll down from swimlane to swimlane
